containers/src/Kokkos_ScatterView.hpp

```diff
@@ -199,35 +211,30 @@ struct ScatterValue;
 
 template <typename ValueType, typename DeviceType>
 struct ScatterValue<ValueType, Kokkos::Experimental::ScatterSum, DeviceType,
-                    Kokkos::Experimental::ScatterNonAtomic>
-    : Sum<ValueType, DeviceType> {
+                    Kokkos::Experimental::ScatterNonAtomic> {
+  ValueType& value;
```

Comment:
Did you mean for this to be public? If so, putting `public:` two lines below makes it look like you didn't intend to do that.